gokart/file_processor.py

```diff
@@ -266,6 +267,22 @@ def dump(self, obj, file):
         dump_obj.to_feather(file.name)
 
 
+class IniFileProcessor(FileProcessor):
```

Comment:
@maronuu How about naming this as ConfigFileProcessor? 
I think this can be apply to other configs like .yaml or .toml

Reply:
I agree with that, changed the name, and supports `.cfg` and `.conf` extensions by `ConfigFileProcessor`.
It follows [this](https://github.com/spotify/luigi/blob/e1d5161765fa505fe131a43608b5476db9c515b2/luigi/configuration/core.py#L29C12-L29C29) in luigi.